public/testem/testem_client.js

```diff
@@ -286,6 +309,18 @@ var Testem = {
     // stringify for clients that only can handle string postMessages (IE <= 10)
     return JSON.stringify(message);
   },
+  removeDuplicateCallbacks: function(evt, callback) {
+    var handlers = this.evtHandlers[evt];
+    var removeIdx = [];
```

Comment:
We seem to have a mixture of `let` and `var` in testem. Which is preferred?

Reply:
I'd go with `let`, as it's the new syntax.

Reply:
for browser destined JavaScript, we should not introduce new JS features. We can in the future, but that would require a major version bump for little value.

It is nice, that testem does still support older platforms, dropping support for them should only occur if there is something high value we gain.

Reply:
Yes, agree. After speaking offline it appears this was destined for browsers, sometimes older browsers, that don't support newer syntaxes. Let's stick with the old stuff.